src/components/common/error/ErrorBar.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { NavLink, useHistory } from 'react-router-dom'
+import { URLS } from '../../../config'
+import { ReactComponent as ErrorInfo } from '../../../assets/icons/misc/errorInfo.svg'
+
+function ErrorBar({ appDetails }) {
+  const history = useHistory()
+    return (
+        <div className="er-2 bw-1 br-4 m-20 fs-13">
+            <div className="bcr-1 pl-12 pr-12 pt-8 pb-8 dc__border-bottom-r2 flex left">
+                <ErrorInfo className="icon-dim-24 mr-8" /> <span className="mr-8">IMAGEPULLBACKOFF:</span>{' '}
+                {!appDetails.ipsAccessProvided ? (
+                    <div>
+                        '{appDetails?.clusterName}' cluster does not have permission to pull container image from ‘
+                        {appDetails?.dockerRegistryId}’ registry.
+                    </div>
+                ) : (
+                    <div>
+                        {appDetails?.clusterName} cluster could not pull container image from{' '}
```

Comment:
can we check for appDetails on single place? so that we don't need to add optional check on multiple place.